lib/skate_web/controllers/report_controller.ex

```diff
@@ -0,0 +1,32 @@
+defmodule SkateWeb.ReportController do
+  @moduledoc """
+  Provides a list of available reports with links to run each one
+  individually and return a CSV.
+  """
+
+  use SkateWeb, :controller
+
+  @spec index(Plug.Conn.t(), map()) :: Plug.Conn.t()
+  def index(conn, _params) do
+    reports =
+      Report.all_reports()
+      |> Enum.map(fn report -> {report.description(), report.short_name()} end)
+
+    conn
+    |> assign(:reports, reports)
+    |> render("index.html", layout: {SkateWeb.LayoutView, "reports.html"})
+  end
+
+  @spec run(Plug.Conn.t(), map()) :: Plug.Conn.t()
+  def run(conn, params) do
+    report = Enum.find(Report.all_reports(), &(&1.short_name() == params["short_name"]))
```

Comment:
the way we're using `find` here makes me think `Report.all_reports` should return a map rather than a list of tuples

Reply:
Hmm, yeah, that would probably be better.